src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days < 3) {
+    return totalCost;
```

Comment:
The initial 'if' condition is unnecessary. Since you return 'totalCost' when 'days' is less than 3, you can omit the 'else' keyword in the subsequent conditions to simplify the code.